app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(text: comment_params[:text], user: current_user)
+
+    respond_to do |format|
+      format.html do
```

Comment:
There should be a pair of routes in the `config/routes.rb` file for creating comments. Th user is not able to comment on a post as there is no route to the resource.